Add riscv64 support.
XXX should probably be reversed to only use rdtsc() where available.

Index: contrib/profiler/Profiler.h
--- contrib/profiler/Profiler.h.orig
+++ contrib/profiler/Profiler.h
@@ -11,7 +11,7 @@
 #undef fastcall
 
 //#define USE_CHRONO
-#if !defined(USE_CHRONO) && (defined(__arm__) || defined(__aarch64__) || defined(_M_AMD64) || defined(_WIN64) || defined(_M_X64))
+#if !defined(USE_CHRONO) && (defined(__arm__) || defined(__aarch64__) || defined(_M_AMD64) || defined(_WIN64) || defined(_M_X64) || defined(__riscv))
 // this isn't optional for __arm__ or x64 builds
 #define USE_CHRONO
 #endif
